cli/ts/utils/interfaces.ts

```diff
@@ -950,7 +950,7 @@ export interface IGetPollData {
   /**
    * Whether the MACI contract's stateAq has been merged by this contract
```

Comment:
then the description here could also be modified to `state root has been merged`